app/controllers/concerns/hal_actions/errors.rb

```diff
@@ -13,6 +13,16 @@ def status
     end
   end
 
+  class NotFound < StandardError
+    def message
+      'Resource not found'
+    end
+
+    def status
+      404
+    end
+  end
+
```

Comment:
yeah, we added this kind of not found handling to the extracted gem. This is fine for now, will have to refactor when we pull this in: https://github.com/PRX/hal_api-rails/pull/1/files